src/database/migrations/20231228020436-isFaculty-forbid-null.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		/**
+		 * Add altering commands here.
+		 *
+		 * Example:
+		 * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+		 */
+		await queryInterface.changeColumn('users', 'isFaculty', {
+			type: Sequelize.DataTypes.BOOLEAN,
+			allowNull: false
+		});
+	},
+
+	down: async (queryInterface, Sequelize) => {
+		/**
+		 * Add reverting commands here.
+		 *
+		 *
```

Comment:
Not so sure why there's an extra line here, would probably be alright if we removed these comments altogether (they come from the template after all)